### ISSUE TITLE
Binary Crossentropy + Accuracy

### ISSUE THREAD

**Issue by cgarciae** [poets-ai/elegy#22]
Use `jax.nn.log_softmax`

**Comment by charlielito** [comment]
@sebasarango1180  implemented Binray Crossentropy already. I'm closing this and opening a new issue just with the Binray Accuracy
